### PR TITLE
Update header bar

### PR DESCRIPTION
**Pull request by sisskind**

### CODE REVIEW

File: modules/ui/sidebar.js [hunk 1]
```diff
@@ -11,6 +11,16 @@ export function uiSidebar(context) {
 
 
     function sidebar(selection) {
+        var labelContainer = selection.append('div').classed('fill-dark',true);
+
+        labelContainer.append('div')
+            .attr('href', '#version')
+            .classed('point hoot_label', true)
+            .attr('height', '60px')
```
Comment (brianhatchl) [inline modules/ui/sidebar.js]: move this prop to css I think

File: img [hunk 2]
```diff
@@ -0,0 +1 @@
+dist/img
```
Comment (brianhatchl) [inline img]: build.js should create this symlink, so remove it

File: css/img [hunk 3]
```diff
@@ -0,0 +1 @@
+../dist/img
```
Comment (brianhatchl) [inline css/img]: build.js should create this symlink, so remove it

**Comment by sisskind** [comment]
@brianhatchl we need to look at the build.js then, because I'm not sure it is 100% doing what we expect

**Pull request merged**